x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/styles.tsx

```diff
@@ -154,3 +154,9 @@ export const StyledTimelineUnifiedDataTable = styled.div.attrs(({ className = ''
     padding-right: 30px;
   }
 `;
+
+export const UnifiedTimelineGlobalStyles = createGlobalStyle`
+  body:has(.timeline-portal-overlay-mask) .unifiedDataTable__cellPopover {
```

Comment:
Thanks for fixing the Popover! We really should have a list of all z-indices in the application somewhere 😂

Reply:
seriously... it is really hard to keep track.

Reply:
I guess the only scenario where I see this would matter is if the user opens the popover, and then tries to open the timeline flyout which would have a `z-index` of `1002`. We don't want the popover to be on top of that new flyout.
But wouldn't the popover close automatically? Is that really a problem we could face?